extra-utils/hdsentinel/autobuild/build

```diff
@@ -0,0 +1,21 @@
+abinfo Detect architecture
+hdsentinelVer="017"
+hdsentinelBaseName="hdsentinel-$hdsentinelVer"
+if [[ "${CROSS:-$ARCH}" = "amd64" ]]; then
+    hdsentinelFileName="$hdsentinelBaseName-x64"
+elif [[ "${CROSS:-$ARCH}" = "i386" ]]; then
+    hdsentinelFileName="$hdsentinelBaseName"
+elif [[ "${CROSS:-$ARCH}" = "armel" ]]; then
+    hdsentinelFileName="$hdsentinelBaseName-arm"
+else
+    exit 1
```

Comment:
It's recommended to set `FAIL_ARCH=!(amd64|i386|armel)` in `defines`, but throwing an error here

Reply:
Cool. Keep `exit 1` here after setting failing arches?